### ISSUE TITLE
Please allow Drill Through to the same report

### ISSUE THREAD

**Issue by shemgp** [shzlw/poli#28]
It would be nice to be able to have a drill through that will refresh only the current report and not go to a different report.  Currently, if I add a drill through and point it to the current report, I still need to click refresh to be able to update the other chart's data.

Thank you developing this software.

**Comment by shzlw** [comment]
Thanks for using Poli. May I know what's the use case for drilling into the current report? Could you try add filters to the current report and use them to control what's displayed in other charts?

**Comment by shemgp** [comment]
Yes, filters can function almost the same way but drill through is more
intuitive since you don't need to find the items in the filter.

Here's a sample use case: first chart on the report displays enrollment
count per semester. At the bottom, i want to show a char with enrollment
per department per selected semester. Instead of adding a semester filter
that will have many items, it would be nice if the use could just click the
sem in the first chart and the 2nd chart will update.  Also, i dont like it
to go to another report as it might confuse the users how to go back.
